src/main/java/com/robertx22/age_of_exile/capability/player/data/DeathFavorData.java

```diff
@@ -29,9 +29,9 @@ public void set(Player p, float f) {
 
         if (rar != getRarity()) {
             if (upped) {
-                p.sendSystemMessage(Chats.FAVOR_UP.locName(getRarity().locName()));
+                p.sendSystemMessage(Chats.FAVOR_UP.locName(getRarity().textFormatting() + getRarity().locName().getString()));
             } else {
-                p.sendSystemMessage(Chats.FAVOR_DOWN.locName(getRarity().locName()));
+                p.sendSystemMessage(Chats.FAVOR_DOWN.locName(getRarity().textFormatting() + getRarity().locName().getString()));
```

Comment:
these two lines have some very weird color format problems, I  have to add the rarity color format as the locName() parameter and reset the following sentence color in Chats.java.